backend/auth/auth.go

```diff
@@ -3,15 +3,15 @@ package auth
 import (
 	"errors"
 	"fmt"
+	"github.com/MicahParks/keyfunc"
+	"github.com/ZPI-2024-25/KubernetesAccessManager/models"
+	"github.com/golang-jwt/jwt/v4"
+	"github.com/joho/godotenv"
```

Comment:
Please no, this kind of sorting doesn't  make sense. I would much prefer seeing builtin imports as a separate list, instead of getting polluted by these github ones

Reply:
GoLand does this automatically.